content/oci/deletableoci_test.go

```diff
@@ -0,0 +1,502 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package oci provides access to an OCI content store.
+// Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc4/image-layout.md
```

Comment:
This needs to be removed, otherwise it will be showed as a duplicated documentation like:
![image](https://github.com/oras-project/oras-go/assets/8113721/4bc52286-759a-4bfc-ab7e-6752a9db541a)

Reply:
removed this line.